cmd/origins/log.go

```diff
@@ -8,16 +8,73 @@ import (
 
 	"github.com/chop-dbhi/origins"
 	"github.com/chop-dbhi/origins/chrono"
+	"github.com/chop-dbhi/origins/storage"
 	"github.com/chop-dbhi/origins/view"
 	"github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
 )
 
+func concatDomains(engine storage.Engine, w origins.Writer, domains []string, since, asof time.Time) int {
+	var (
+		err      error
+		n, count int
+		log      *view.Log
+	)
+
+	// Output facts for domains in order.
```

Comment:
You have the same comment for both concatenate and merge -- maybe something like "output facts ordered by domain" for concatenate and "... ordered by transaction commit time" for merge (though technically now it's by transaction id...).
I don't have anything else, the code looks good.